Directory.Packages.props

```diff
@@ -4,6 +4,8 @@
     <PackageVersion Include="Expecto" Version="[10.0, 11.0)" />
     <PackageVersion Include="FSharp.Core" Version="[7.0.200,)" />
     <PackageVersion Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+    <PackageVersion Include="Microsoft.Testing.Extensions.Telemetry" Version="1.0.1" />
```

Comment:
This is not mandatory but would be greatly appreciated as it helps us understand usage and justify further work to management.